src/accelerate/hooks.py

```diff
@@ -266,13 +276,13 @@ def init_hook(self, module):
             for name, _ in named_module_tensors(
                 module, include_buffers=self.offload_buffers, recurse=self.place_submodules, remove_non_persistent=True
             ):
-                set_module_tensor_to_device(module, name, "meta")
+                set_module_tensor_to_device(module, name, "meta", tied_params_map=tied_params_map)
```

Comment:
We probably don't need the `tied_params_map` since we are sending them to the `meta` device.